frontend/src/pages/about/PagePublications.vue

```diff
@@ -6,7 +6,13 @@
 
 <template>
   <AppSection>
-    <AppHeading>Publications</AppHeading>
+    <AppHeading>Monarch-Related Publications</AppHeading>
```

Comment:
Probably unnecessary to say "Monarch-Related", since it's explained below, and more specific of a header than other pages. Not a big deal though.

Reply:
This was just the requested change from #323

Reply:
I've pushed back on this but you can merge without it.